tests/JavaScriptTest.js

```diff
@@ -246,7 +246,7 @@ function testUnicode() {
   MyGame.Example.Monster.addTestarrayofstring(fbb, testarrayofstringOffset);
   MyGame.Example.Monster.addTestarrayoftables(fbb, testarrayoftablesOffset);
   MyGame.Example.Monster.addName(fbb, name);
-  MyGame.Example.Monster.finishMonsterBuffer(fbb, MyGame.Example.Monster.endMonster(fbb));
+  MyGame.Example.Monster.finishSizePrefixedMonsterBuffer(fbb, MyGame.Example.Monster.endMonster(fbb));
```

Comment:
Where does `finishSizePrefixedMonsterBuffer` come from? Is there a code-gen aspect to this PR?

Reply:
Yes it's supposed to be added by code-gen, I'm still figuring that part out. Sorry for not being clear about this in the PR description.

Reply:
I think I got it right it with https://github.com/google/flatbuffers/pull/5326/commits/ba77cc3eac427859d67515eba94674815b90e5dd

Reply:
Seems I got through it now, implemented the code-gen changes and now have a working test case.